src/scenes/cheatSheet.js

```diff
@@ -1,4 +1,25 @@
-const cheatSheet = function()
-{
-    background(0, 255, 0);
-}
\ No newline at end of file
+let cheatSheetImg;
+let cheatSheetBtnImg;
+let cheatSheetBtn;
+const cheatSheet = {
+    "setup": function()
+    {
+		cheatSheetImg = loadImage("assets/textures/CheatSheet.png",
+			() => cheatSheetImg = scaleNearestNeighbor(cheatSheetImg, width, height));
```

Comment:
I'm worried about how long this will take to load on high-resolution screens, such as the iPads that the students might be using. At the current resolution (640x480), it's taking more than 2 seconds for me, which feels glacial; once we implement making this game full-screen it will likely take even longer.

I tried scaling the image by passing in the `width` and `height` in the call to `image()` on line 23 below and while it was significantly faster, I could see why you needed to use this scaling method - the image was quite blurry.

What I'm wondering is if we might be able to improve performance here by scaling up the original image, so that when we're resizing it, we're always sizing down?

Reply:
Alright, I'll do some testing and see if I can optimize it. I didn't run into any issues myself though. I run it through apache on linux. Something I might do is write a function in index through setup or preload to load in everything so that we aren't going back to the server to fetch assets that really should be in memory. This would then front load all the wait time, but would leave a smoother experience once everything is loaded.

Reply:
That's a good point, it could have been my local environment slowing things down - I tested this using both the 'Live Preview' and 'Live Server' extensions on VS Code, which I'm not sure what goes on under the hood of those things. When I tried to load the page in Firefox via Live Server my browser warned me about high load and asked if I wanted to stop the processing going on that tab; it didn't even finish loading the main menu 😥